### ISSUE TITLE
Using Worldcat data for publications

### ISSUE THREAD

**Issue by mbelhu** [Worlding-Public-Cultures/wpc-website#13]
Currently the publication content type has a `field_worldcat_link` where the WorldCat link of a publication can be entered. Would it be possible to reuse WorldCat data so researchers do not have to enter all the bibliographic information if it's in WorldCat already?
Ideally researchers would have the option of either entering the WorldCat link or filling in the form when the publication is not in WorldCat.

**Comment by mbelhu** [comment]
I fixed the entity reference linkage of the publication content type in exhibitions, academic courses and public events since I got it wrong. This will go in the next configuration push.

**Comment by natuk** [comment]
I do not think this automatic data fetching is possible with Drupal. leaving open to investigate modules further.
